pkgs/by-name/mi/microsocks/package.nix

```
@@ -0,0 +1,28 @@
+{
+  stdenv,
+  fetchFromGitHub,
+  lib
+}:
+stdenv.mkDerivation rec {
+  pname = "microsocks";
+  version = "1.0.4";
+  src = fetchFromGitHub {
+    owner = "rofl0r";
+    repo = "microsocks";
+    rev = "v${version}";
+    fetchSubmodules = true;
+    sha256 = "sha256-cB2XMWjoZ1zLAmAfl/nqjdOyBDKZ+xtlEmqsZxjnFn0=";
+  };
+
+  installPhase = ''
+    install -Dm 755 microsocks -t $out/bin/
+  '';
+
+  meta = with lib; {
+    homepage = "https://github.com/rofl0r/microsocks";
+    description = "tiny, portable SOCKS5 server with very moderate resource usage ";
```

See: [https://github.com/NixOS/nixpkgs/blob/3551c607f454ee855e54db5686464c8dcf83ceae/doc/stdenv/meta.chapter.md#description-var-meta-description](https://github.com/NixOS/nixpkgs/blob/3551c607f454ee855e54db5686464c8dcf83ceae/doc/stdenv/meta.chapter.md#description-var-meta-description)

```suggestion
    description = "Tiny, portable SOCKS5 server with very moderate resource usage";
```